niaid.bionimbus.org/manifest.json

```diff
@@ -27,6 +27,14 @@
   "jupyterhub": {
     "enabled": "yes"
   },
+  "arranger": {
+    "project_id": "niaid_0",
+    "auth_filter_field": "auth_resource_path",
+    "auth_filter_node_type": "subject"
+  },
+  "tube": {
+    "es_index_name": "niaid"
```

Comment:
the index name should also be versioned
and use `gen3 es alias niaid_0 niaid` to create `niaid` alias